### ISSUE TITLE
Unnecesary files/folders in release android build

### ISSUE THREAD

**Issue by zZHorizonZz** [dotnet/maui#14437]
### Description

I have the same issue as described in this issue #13785. After closer inspection of the .apk file. I noticed that the is a lib folder that contains armeabi-v7a,x86,x86_64,arm64-v8a architecture folder which does seem unnecessary to me since to my knowledge you should be able to customize on google play what architecture gets what .apk file. I don't know if I'm building the .apk file right but I didn't find any after building the release other .apk files.

On the left you can see my testing application on the right you can see an application from [here](https://github.com/jonathanpeppers/maui-profiling/blob/main/apks/maui.rtm/release-aot/com.companyname.hellomaui-Signed.apk) mentioned [here](https://github.com/dotnet/maui/issues/13785#issuecomment-1464522995)
![image](https://user-images.githubusercontent.com/60035933/230360719-da829ee2-c763-4eee-b21e-d3ffed25dfa7.png)

### Steps to Reproduce

Currently, I'm using the release build of the app from the bin/android folder.

### Link to public reproduction project repository

None

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

All Android versions

**Comment by jsuarezruiz** [comment]
@jonathanpeppers Thoughts?

**Comment by zZHorizonZz** [comment]
> @zZHorizonZz you actually submit [Android App Bundles](https://developer.android.com/guide/app-bundle/app-bundle-format) to Google Play (`.aab` files). Google Play can then split up your app bundle and deliver device-specific `.apk` files. You can set `RuntimeIdentifier` or `RuntimeIdentifiers` in your project to select how many architectures you are compiling for.
> 
> Can you rephrase a bit what your problem is? I'm not seeing what we need to change, or is it just a question?

I didn't know about this feature. I was assuming you need a specific .apk for each architecture or just you need to provide .apk with all architectures included. Thank you for the explanation.
